addons/website_slides/models/slide_slide.py

```diff
@@ -183,6 +183,7 @@ class Slide(models.Model):
     image_google_url = fields.Char('Image Link', related='url', readonly=False,
         help="Link of the image (we currently only support Google Drive as source)")
     # content - documents
+    slide_icon = fields.Char('Slide Icon', compute='_compute_slide_icon')
```

Comment:
"slide_icon" would refer more to an actual image or unicode character to define the icon.

Here it's more the fa-class, maybe "slide_icon_class" ?